include/storage-area.h

```diff
@@ -12,4 +12,5 @@ extern OrthancPluginErrorCode StorageReadRangeCallback(OrthancPluginMemoryBuffer
                                                        const char *uuid,
                                                        OrthancPluginContentType type,
                                                        uint64_t rangeStart);
-extern OrthancPluginErrorCode StorageRemoveCallback(const char *uuid, OrthancPluginContentType type);
\ No newline at end of file
+extern OrthancPluginErrorCode StorageRemoveCallback(const char *uuid, OrthancPluginContentType type);
+extern OrthancPluginErrorCode OnStoredInstanceCallback(const OrthancPluginDicomInstance *instance, const char *instanceId);
```

Comment:
We should move this to plugin.cpp or its own file (not really part of the storage-area)